cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -53,4 +69,6 @@ package object cwl extends TypeAliases {
 
   type Yaml = String
```

Comment:
You all know my take on this. Unless it's a type alias to make cats happy (which I don't think either this one or the new one below are) I'd prefer to see it as a wrapped case class even if that makes @mcovarr grumpy having to type a few extra chars before his autocomplete works.